autoarray/operators/convolver.py

```diff
@@ -555,3 +555,54 @@ def convolve_matrix_jit(
                         )
 
         return blurred_mapping_matrix
+
+    def summed_convolved_array_from(self, image):
+        """For a given 1D array and blurring array, convolve the two using this convolver.
+
+        Parameters
+        ----------
+        image
+            1D array of the values which are to be blurred with the convolver's PSF.
+        blurring_image
+            1D array of the blurring values which blur into the array after PSF convolution.
+        """
+
+        summed_convolved_array = self.convolve_no_blurring_jit(
+            image_1d_array=image.binned.slim,
+            image_frame_1d_indexes=self.image_frame_1d_indexes,
+            image_frame_1d_kernels=self.image_frame_1d_kernels,
+            image_frame_1d_lengths=self.image_frame_1d_lengths,
+        )
+
+        return Array2D(values=summed_convolved_array, mask=self.mask.derive_mask.sub_1)
+
+    @staticmethod
+    #    @numba_util.jit()
+    def summed_convolved_array_jit(
+        image_1d_array,
+        image_frame_1d_indexes,
+        image_frame_1d_kernels,
+        image_frame_1d_lengths,
+    ):
+
+        summed_convolved_array = np.zeros(image_1d_array.shape)
+
+        for image_1d_index in range(len(image_1d_array)):
+
+            print()
+
+            frame_1d_indexes = image_frame_1d_indexes[image_1d_index]
+            frame_1d_kernel = image_frame_1d_kernels[image_1d_index]
+            frame_1d_length = image_frame_1d_lengths[image_1d_index]

```

Comment:
I think you could use zip rather than indexing for these four collections

Reply:
I think this was chosen to help numba.